CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   * `-flusher.concurrent-flushes` for number of concurrent flushes.
   * `-flusher.flush-op-timeout` is duration after which a flush should timeout.
 * [ENHANCEMENT] Better re-use of connections to DynamoDB and S3. #2268
+* [ENHANCEMENT] Fewer goroutines for queries. #2280
```

Comment:
Could you me more explicit, please? Maybe something like `Reduce number of goroutines used while executing a single index query` ?

Reply:
Sure.  TBH I'm unclear exactly where this helps; I was chasing a situation where one query creates 7,000 goroutines but now I'm fairly sure this isn't the fix.  But yes, your wording is clearer.